fontc_crater/src/main.rs

```diff
@@ -159,9 +159,10 @@ impl<T, E> Default for Results<T, E> {
 impl From<LoadRepoError> for SkipReason {
     fn from(value: LoadRepoError) -> Self {
         match value {
-            LoadRepoError::Io(_) | LoadRepoError::GitFail(_) | LoadRepoError::NoCommit { .. } => {
-                SkipReason::GitFail
-            }
+            LoadRepoError::Io(_)
+            | LoadRepoError::GitFail(_)
+            | LoadRepoError::NoCommit { .. }
+            | LoadRepoError::MissingAuth => SkipReason::GitFail,
```

Comment:
If skip reasons surface in the UI or logs I suggest having one specifically for missing auth

Reply:
actually I think SkipReason is now completely unused, can delete in a followup.